README.md

```diff
@@ -1 +1,15 @@
-# bitrise-step-android-lint
+# Android Lint
```

Comment:
use our step template's README: https://github.com/bitrise-steplib/step-template/blob/master/README.md
until we introduce our new REAGME guide.